Cargo.toml

```diff
@@ -14,19 +14,18 @@ edition = "2018"
 base64 = "0.10"
 chrono = "0.4"
 http = "0.1"
-hyper = {version = "0.12", default-features = false}
-hyper-rustls = "0.17"
+hyper = {version = "0.13.0-alpha.4", features = ["unstable-stream"]}
+hyper-rustls = "=0.18.0-alpha.2"
 itertools = "0.8"
 log = "0.3"
 rustls = "0.16"
 serde = "1.0"
 serde_json = "1.0"
 serde_derive = "1.0"
 url = "1"
-futures = "0.1"
-tokio-threadpool = "0.1"
-tokio = "0.1"
-tokio-timer = "0.2"
+futures-preview = "=0.3.0-alpha.19"
```

Comment:
Hi @ggriffiniii atleast `futures-preview` are no longer alpha so I guess you could use `futures 0.3.1` instead.

Reply:
We need to align the dependency versions with the versions hyper is using so that traits match. Once hyper releases a new version we can update accordingly.

Reply:
https://github.com/hyperium/hyper/pull/2007 is the PR for hyper